spring-cloud-stream/src/test/java/org/springframework/cloud/stream/binder/integration/SpringIntegrationChannelBinder.java

```diff
@@ -128,10 +128,6 @@ public void setMessageSourceDelegate(MessageSource<?> messageSourceDelegate) {
 		return this.lastError;
 	}
 
```

Comment:
Should this PR include moving this binder to `spring-cloud-stream-test` and having a subclass of `AbstractBinderTests` for this binder there?

Or is that considered a separate effort?

Reply:
I would do it as another effort, perhaps right after this one is merged. It's too big as it is